compatibility_server/pip_checker.py

```diff
@@ -431,13 +436,20 @@ def _install(self, container: docker.models.containers.Container):
             # sys	0m0.357s
             client_repo_directory = self._clone_repo(
                 container, CLIENTLIBS_GITHUB_URL)
+        elif any(GITHUB_OPENCENSUS_PREFIX in pkg for pkg in self._packages):
+            client_repo_directory = self._clone_repo(
```

Comment:
This is problematic when the `self._packages` contains both opencensus and google_cloud_python packages, as only `client_repo_directory` variable is used in here. Not sure whether fixing this will fix the kokoro timeout issue, but it's definitely something we need to fix.